src/store/modules/orders/mutation-types.ts

```diff
@@ -0,0 +1,5 @@
+export const SN_ORDERS = 'orders'
+export const ORDERS_CURRENT_UPDATED = SN_ORDERS + '/CURRENT_UPDATED'
+export const OPEN_ORDERS_INITIAL = SN_ORDERS + '/ORDERS_INITIAL'
+export const ORDERS_PACKED_INITIAL = SN_ORDERS + '/PACKED_INITIAL'
```

Comment:
`ORDER_PACKED_UPDATED`  
In any case whether we do it initially or on any other action. We are updating the packed orders.

Reply:
Changed name to `ORDER_PLACED_UPDATED`